.semaphore/semaphore.yml

```diff
@@ -79,7 +79,7 @@ blocks:
         - name: Create
           commands:
             - gem install rails -v $RAILS_VERSION --no-document
-            - make create OPTIONS="--api"
+            - make create_api OPTIONS="--api"
```

Comment:
since we have `create_api` already, should we moving the `--api` to Makefile too 🤔

Reply:
Added in [16f5f5f](https://github.com/nimblehq/rails-templates/pull/237/commits/16f5f5fc5a6332819f74436dd7719b53c97e5d97)